api/clientset/v1beta1/topology_test.go

```diff
@@ -78,7 +80,26 @@ func setUp(t *testing.T) (*Clientset, *restfake.RESTClient) {
 	}
 	objs := []runtime.Object{obj1, obj2}
 	cs.restClient = fakeClient
-	cs.dInterface = dynamicfake.NewSimpleDynamicClient(scheme.Scheme, objs...).Resource(gvr)
+	f := dynamicfake.NewSimpleDynamicClient(scheme.Scheme, objs...)
+	// Add handler for Update call.
+	f.PrependReactor("update", "*", func(action ktest.Action) (bool, runtime.Object, error) {
+		uAction, ok := action.(ktest.UpdateAction)
+		if !ok {
+			return false, nil, nil
+		}
+		fmt.Println(action)
```

Comment:
Add context to what is being printed for looking at the test logs

(here and with tObj)